package.json

```diff
@@ -5,28 +5,40 @@
     "build": "npm run clean && NODE_ENV=production webpack",
     "build:dev": "npm run clean && NODE_ENV=development webpack",
     "clean": "rm -rf public/dist",
-    "test": "jest",
+    "test": "jest --watch",
     "test:ci": "NODE_ENV=test && jest"
   },
   "babel": {
     "presets": [
       "@dosomething/babel-config"
     ]
   },
+  "jest": {
+    "moduleNameMapper": {
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/resources/assets/__mocks__/fileMock.js",
+      "\\.(css|scss)$": "identity-obj-proxy"
+    }
+  },
   "dependencies": {
     "@dosomething/forge": "^6.7.4",
     "@dosomething/gateway": "^1.1.5",
     "classnames": "^2.2.5",
     "dosomething-modal": "^0.3.0",
     "lodash": "^4.17.4",
     "react": "^15.4.2",
-    "react-dom": "^15.4.2"
+    "react-dom": "^15.4.2",
+    "sinon": "^3.3.0"
   },
   "devDependencies": {
     "@dosomething/babel-config": "^1.0.0",
     "@dosomething/webpack-config": "^2.0.4",
     "babel-jest": "^21.0.2",
+    "enzyme": "^2.9.1",
+    "enzyme-to-json": "^2.0.0",
+    "identity-obj-proxy": "^3.0.0",
     "jest": "^21.0.2",
+    "react-dom": "^15.6.1",
```

Comment:
Looks like this got duplicated from `dependencies`. Let's remove this line.

Reply:
oops! Thanks for catching that - deleted! :)